src/components/Anchor/index.js

```diff
@@ -0,0 +1,22 @@
+import React, {PropTypes} from 'react'
+
+const Anchor = ({
+  url,
+  children,
+  isSeparateTab = false,
+}) => (
+  <a
+    href={url}
+    target={isSeparateTab ? '_blank' : '_self'}
+    className='blue'
+  >
+    {children}
+  </a>
+)
+
+Anchor.propTypes = {
+  url: PropTypes.string.isRequired,
+  children: PropTypes.node,
+}
+
+export default Anchor
```

Comment:
Many of these top level generic components will be removed from this project and `yarn install`ed from `egghead-ui` once that project is ready for it.